driver/src/orderbook/streamed/state.rs

```diff
@@ -323,17 +323,122 @@ mod tests {
     use super::*;
     use crate::models::AccountState;
 
+    macro_rules! apply_event {
+        (to $state:ident in batch $batch:expr, list token $token:expr) => {
```

Comment:
ubernit: 
```suggestion
        (to $state:ident for batch $batch:expr; TokenListing token $token:expr) => {
```
- I would `s/in batch/for batch/` as it sounds a little nicer for me.
- I slightly prefer a semi-colon separating the batch value and the event information, this is just so there is a clearer separation between the state and batch information and the event parameters. Also since commas are used elsewhere in the macro, it makes it easier at a quick glance to separate the two parts.
- I would also slightly prefer to use the event name instead of a verb. For me this just removes some of the cognitive load in mapping event keywords to actual events.

Reply:
I like these suggestions, especially the last one!